packages/ui/src/components/MenuV2/components/Item.tsx

```diff
@@ -142,7 +148,10 @@ const Item = forwardRef<HTMLElement, ItemProps>(
     },
     ref,
   ) => {
-    const { hideOnClickItem, setIsVisible } = useMenu()
+    const { hideOnClickItem, setIsVisible, itemsList } = useMenu()
+
+    const innerRef = useRef<HTMLButtonElement>(null)
+    const computedRef = ref ?? innerRef
```

Comment:
ref is not an HTMLButtonElement that why you need to cast here ?

https://github.com/scaleway/ultraviolet/pull/5255/files#diff-d4c7de53cc545c6b7fe43a827e9d37e900fb41229803b99988b0c67bc8e40dc6R175